docs/sources/observability-as-code/provision-resources/intro-git-sync.md

```diff
@@ -54,31 +54,33 @@ With Git Sync, you can make changes in your provisioned files in GitHub and see
 
 Git Sync is under development and the following limitations apply:
 
+- You can only sync dashboards and folders. 
+  - If you're using Git Sync in Grafana OSS and Grafana Enterprise, some resources might be in an incompatible data format and can't be synced. 
+  - If you're using Git Sync in Grafana Cloud, Git Sync only works with specific folders for the moment. Full-instance sync is not currently supported.
```

Comment:
Full-instance sync will be also blocked/removed in Grafana OSS, and Grafana Enterprise. So, maybe we should change the narrative, and not calling this a limitation, but as part of how the synchronization works. Git Sync will create a folder for all the synchronized resources to live under. That's probably part of the synchronization part of the docs, and not in the limitations. 

We could say that you can have unmanaged resources outside that folder, and having a mix of managed / unmanaged is supported and normal.